kivymd/uix/card.py

```diff
@@ -624,8 +624,8 @@ class MDCard(
     BackgroundColorBehavior,
     RectangularElevationBehavior,
     FocusBehavior,
-    BoxLayout,
     RectangularRippleBehavior,
+    BoxLayout,
```

Comment:
It won't work. I have already closed the pull requests associated with this fix. Cards will not scroll in the list

Reply:
That's right. I'll go deeper into this issue